package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "@babel/preset-typescript": "^7.9.0",
     "@sls-next/lambda-at-edge": "file:packages/lambda-at-edge",
+    "@sls-next/s3-static-assets": "^1.0.1",
```

Comment:
@danielcondemarin should i remove this?? I needed it to run the tests but i gathered that this is handled elsewhere?

Reply:
That doesn't look right, yeah remove it please.

Reply:
i have removed it. any other changes you'd like?

Reply:
Nope, thanks. I'm now tidying up some of the tests and doing a bit of manual testing.